### ISSUE TITLE
The plugin fails with Gradle configuration cache enabled

### ISSUE THREAD

**Issue by sergeykad** [bjornvester/xjc-gradle-plugin#20]
There is a new Gradle feature - configuration caching: https://docs.gradle.org/current/userguide/configuration_cache.html
This plugin is not compatible with it and fails if the caching feature is enabled.

The following error is printed:
```
* What went wrong:
Some problems were found with the configuration of task ':broker-model:xjc' (type 'XjcTask').
  - Type 'com.github.bjornvester.xjc.XjcTask' property 'xjcBindConfiguration' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'xjcBindConfiguration'.
      2. Mark property 'xjcBindConfiguration' as optional.
    
    Please refer to https://docs.gradle.org/7.1/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'com.github.bjornvester.xjc.XjcTask' property 'xjcConfiguration' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'xjcConfiguration'.
      2. Mark property 'xjcConfiguration' as optional.
    
    Please refer to https://docs.gradle.org/7.1/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'com.github.bjornvester.xjc.XjcTask' property 'xjcPluginsConfiguration' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'xjcPluginsConfiguration'.
      2. Mark property 'xjcPluginsConfiguration' as optional.
    
    Please refer to https://docs.gradle.org/7.1/userguide/validation_problems.html#value_not_set for more details about this problem.
```

**Comment by belovaf** [comment]
Imade PR which fix this issue.
https://github.com/bjornvester/xjc-gradle-plugin/pull/22

**Comment by davemaul** [comment]
@bjornvester it'd be great if you could merge the PR. :)

**Comment by ChristianCiach** [comment]
@bjornvester You don't seem to react to any issue or PR in this repo, ever. This is your good right, but I recommend to disable the Issues- and PR tabs in your Github project if you have no interest in interacting with the community. Otherwise people just waste their time.

**Comment by sergeykad** [comment]
Closing, since the issue was resolved in [v1.7.0](https://github.com/bjornvester/xjc-gradle-plugin/releases/tag/v1.7.0)
